Makefile

```diff
@@ -42,31 +42,31 @@ integration-postgres: .env ## Runs postgres integration tests with py38.
 
 .PHONY: integration-postgres-fail-fast
 integration-postgres-fail-fast: .env ## Runs postgres integration tests with py38 in "fail fast" mode.
-	$(DOCKER_CMD) tox -e py38-postgres -- -x -nauto
+	$(DOCKER_CMD) tox -e py38-postgres -- -x
```

Comment:
what was -nauto?

Reply:
It is an argument for a pytest plugin that is used to split up tests and run in separate processes ([docs](https://github.com/pytest-dev/pytest-xdist#speed-up-test-runs-by-sending-tests-to-multiple-cpus)). It does not play nicely with the the `-x` arg (exit instantly on first error or failed test).

Reply:
I take that back, this is fine! It doesn't work with `-s` (turns of stdout capturing) I'll clean up these changes, they aren't related anyway.